### PR TITLE
feat: updated mutation with email

### PR DESCRIPTION
**Pull request by sruddy**

### CODE REVIEW

File: src/react/boosts/payments/BoostPayFormEth.tsx [hunk 1]
```diff
@@ -60,14 +73,18 @@ export class BoostPayFormEth extends React.Component<BoostPayFormEthProps, Boost
...
-          </BoostPayFormContain>*/}
+          <BoostUserInfoForm>
+            <label>Email (optional)</label>
+            <BoostInput
+              valid={this.state.validEmail}
+              id="email"
+              name="email"
+              type="email"
+              maxLength={254}
+              onChange={() => this.handleOnChange(event)}
```
Comment (nickreynolds) [inline src/react/boosts/payments/BoostPayFormEth.tsx]: definitely ok for now, but eventually we should do that fancy thing where it only validates once the user stops typing. I saw a really slick way of doing this using observables a little while back, I'll see if I can find it again (it honestly might've been a Consensys dev session)

**Comment by nickreynolds** [comment]
should this be merged into master or you want it into this branch?

**Comment by sruddy** [comment]
https://github.com/joincivil/civil-sdk/pull/52 needs to be merged first.

File: src/react/boosts/payments/BoostPayFormEth.tsx (hunk 1 above)
Comment (sruddy) [reply src/react/boosts/payments/BoostPayFormEth.tsx]: Agreed I'm not a fan of onChange validation but it works for now

**Pull request merged**